keyboards/sol/rev1/rev1.h

```diff
@@ -0,0 +1,70 @@
+#ifndef REV1_H
+#define REV1_CONFIG_H
+
+#include "../sol.h"
```

Comment:
This is in the include path for make, so shouldn't need to specify the path
```suggestion
#include "sol.h"
```